packages/sitecore-jss-nextjs/src/utils.ts

```diff
@@ -30,6 +30,15 @@ export const getPublicUrl = (): string => {
   return url.toString().replace(/\/$/, '');
 };
 
+/**
+ * Get hostname from from the provided URL.
+ * @param {Array} [urls] - The URL to parse
+ */
+export const getHostName = (urls: [string]): Array<string> => {
```

Comment:
I'd suggest accepting a single URL string. We should also add some unit tests for this.